ui/Screen/Project/Source.svelte

```diff
@@ -188,11 +188,7 @@
           <div class="repo-stat-item">
             <Icon.Commit />
             <Text style="margin: 0 8px;">
-              <!-- Nb. We don't `use:link` here because it doesn't work due to
-              some reactivity bug. -->
-              <a
-                href={`#${path.projectCommits(project.id, $currentRevision)}`}
-                data-cy="commits-button">
```

Comment:
Why is this removed for this change? :thinking:

Reply:
It's different on master now